docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -377,27 +377,27 @@ This keeps other provisioners available to handle user-initiated jobs.
 
 #### Setup
 
-1) Create a provisioner key with a prebuild tag (e.g., `is_prebuild=true`).
-Provisioner keys are org-scoped and their tags are inferred automatically by provisioner daemons that use the key.
-**Note:** `coder_workspace_tags` are cumulative, so if your template already defines provisioner tags, you will need to create the provisioner key with the same tags plus the `is_prebuild=true` tag so that prebuild jobs correctly match the dedicated prebuild pool.
-See [Scoped Key](../../provisioners/index.md#scoped-key-recommended) for instructions on how to create a provisioner key.
-
-2) Deploy a separate provisioner pool using that key (for example, via the [Helm coder-provisioner chart](https://github.com/coder/coder/pkgs/container/chart%2Fcoder-provisioner)).
-Daemons in this pool will only execute jobs that include all of the tags specified in their provisioner key.
-See [External provisioners](../../provisioners/index.md) for environment-specific deployment examples.
-
-3) Update the template to conditionally add the prebuild tag for prebuild jobs.
-
-```hcl
-data "coder_workspace_tags" "prebuilds" {
-  count = data.coder_workspace_owner.me.name == "prebuilds" ? 1 : 0
-  tags = {
-    "is_prebuild" = "true"
-  }
-}
-```
+1. Create a provisioner key with a prebuild tag (e.g., `is_prebuild=true`).
+    Provisioner keys are org-scoped and their tags are inferred automatically by provisioner daemons that use the key.
+    **Note:** `coder_workspace_tags` are cumulative, so if your template already defines provisioner tags, you will need to create the provisioner key with the same tags plus the `is_prebuild=true` tag so that prebuild jobs correctly match the dedicated prebuild pool.
+    See [Scoped Key](../../provisioners/index.md#scoped-key-recommended) for instructions on how to create a provisioner key.
+
+1. Deploy a separate provisioner pool using that key (for example, via the [Helm coder-provisioner chart](https://github.com/coder/coder/pkgs/container/chart%2Fcoder-provisioner)).
+    Daemons in this pool will only execute jobs that include all of the tags specified in their provisioner key.
+    See [External provisioners](../../provisioners/index.md) for environment-specific deployment examples.
+
+1. Update the template to conditionally add the prebuild tag for prebuild jobs.
+
+    ```hcl
+    data "coder_workspace_tags" "prebuilds" {
+      count = data.coder_workspace_owner.me.name == "prebuilds" ? 1 : 0
+      tags = {
+        "is_prebuild" = "true"
+      }
+    }
+    ```
 
-Prebuild workspaces are a special type of workspace owned by the system user `prebuilds`.
+Prebuild workspaces are a special type of workspace owned by the system user `prebuild`.
```

Comment:
@matifali this value should not have been changed, the system user is `prebuilds` (with an 's')